dbt/task/init.py

```diff
@@ -25,8 +25,8 @@
 
 # uncomment below and add real repositories to add dependencies to this project
 #repositories:
-#  - "git@github.com:analyst-collective/repo-name-1"
-#  - "git@github.com:analyst-collective/repo-name-2"
+#  - "git@github.com:fishtown-analytics/repo-name-1"
```

Comment:
i actually though about that for a second... figured the use case was more `fishtown-analytics/snowplow` than `your-org/your-package`. But sure, np